pkg/controller/efs/filesystem/setup.go

```diff
@@ -2,9 +2,13 @@ package filesystem
 
 import (
 	"context"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/pkg/errors"
+	"k8s.io/utils/ptr"
 
 	"github.com/aws/aws-sdk-go/aws"
 	svcsdk "github.com/aws/aws-sdk-go/service/efs"
+	svcsdkapi "github.com/aws/aws-sdk-go/service/efs/efsiface"
 	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
 	"github.com/crossplane/crossplane-runtime/pkg/connection"
 	"github.com/crossplane/crossplane-runtime/pkg/controller"
```

Comment:
The linter requires Go imports to be in a specific order (defined in `.golangci-lint.yml`) separated by empty lines:

* Go standard packages
* Third-party packages
* `github.com/crossplane-contrib/provider-aws/*`